odoo/addons/base/ir/ir_model_view.xml

```diff
@@ -279,7 +279,7 @@
                                 </group>
                                 <group string="Advanced Properties" groups="base.group_no_one">
                                     <field name="related"/>
-                                    <field name="depends" attrs="{'required': [('compute', '!=', False)] }"/>
+                                    <field name="depends" attrs="{'required': [('compute', '!=', False), ('compute', '!=', '')] }"/>
```

Comment:
`attrs="{'required': [('compute', 'not in', [False, ''])] }"` should work as well.